### ISSUE TITLE
ctx.message not contains optional reply_markup property

### ISSUE THREAD

**Issue by agnoam** [telegraf/telegraf#791]
## Context

Please provide any relevant information about your setup. This is important in case the issue is not reproducible except for under certain conditions.

* Telegraf.js Version: `^3.32.0`
* Node.js Version: `v12.13.0`
* Operating System: Windows

## Expected Behavior
I expect to find, `reply_markup` property inside `ctx.message` as given by the package. but it is not declared in the declaration file.

````
function containsInlineButtons(ctx: ContextMessageUpdate): boolean {
    if(ctx.message) {
        if(ctx.message['reply_markup']) {
            const markup = ctx.message['reply_markup'];
            if(markup['inline_keyboard'].length > 0) {
                return true;
            }
            return false;
        }
        return false;
    }
    return false;
}
````

## Current Behavior
The package does not contain the optional `reply_markup` property inside `ctx.message`. (`tt.IncomingMessage`)

**Comment by sjinks** [comment]
The bug should probably be filed against https://github.com/sergeysova/telegram-typings repository, as this is where `Message` interface is defined.

As a workaround, you can use something like this:

```typescript
function containsInlineButtons(ctx: ContextMessageUpdate): boolean {
    if (ctx.message) {
        const markup: InlineKeyboardMarkup | undefined = (ctx.message as any).reply_markup;
        return markup && markup.inline_keyboard.length > 0;
    }

    return false;
}
```

**Comment by wojpawlik** [comment]
Already fixed on `develop` (by switching to https://github.com/KnorpelSenf/typegram), will be published in 4.0.
